nextjs-13/package.json

```diff
@@ -6,15 +6,31 @@
     "dev": "next dev -p 3300",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage"
   },
   "dependencies": {
+    "@faker-js/faker": "^8.0.1",
+    "@prisma/client": "^4.14.1",
+    "@testing-library/jest-dom": "^5.16.5",
     "@types/node": "20.2.1",
+    "@types/passport": "^1.0.12",
+    "@types/passport-google-oauth20": "^2.0.11",
     "@types/react": "18.2.6",
     "@types/react-dom": "18.2.4",
     "eslint": "8.40.0",
     "eslint-config-next": "13.4.3",
+    "iron-session": "^6.3.1",
+    "jest": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0",
+    "jest-mock-extended": "^3.0.4",
     "next": "13.4.3",
+    "next-connect": "0.13.0",
```

Comment:
This package is now outdated as there is a newer 1.0.0 version that has been in the works and released: https://github.com/hoangvvo/next-connect/releases/tag/v1.0.0